components/status/StatusContent.vue

```diff
@@ -34,6 +71,7 @@ const hideAllMedia = computed(
 <template>
   <div
     space-y-3
+    my-4
```

Comment:
I think this is having an unintended effect on every post
<img width="616" alt="image" src="https://github.com/elk-zone/elk/assets/583075/a096feb8-3649-4bdb-ae73-e7f88234ae53">

Reply:
Oops! I'll revert the change.